crates/circuits/mod-builder/src/README.md

```diff
@@ -1,4 +1,35 @@
-### Example usage
+## Notes on flags and setup
```

Comment:
https://linear.app/intrinsictech/issue/INT-2907/simplyunify-the-expr-builder-to-all-have-setup